autopush/main.py

```diff
@@ -219,6 +220,14 @@ def _parse_endpoint(sysargs):
                         type=int, default=8082, env_var="PORT")
     parser.add_argument('--cors', help='Allow CORS PUTs for update.',
                         type=bool, default=False, env_var='ALLOW_CORS')
+    parser.add_argument('--s3_bucket', help='S3 Bucket for SenderIDs',
+                        type=str, default=DEFAULT_BUCKET,
+                        env_var='S3-BUCKET')
+    parser.add_argument('--senderid_expry', help='Cache expry for senderIDs',
+                        type=int, default=SENDERID_EXPRY,
+                        env_var='SENDERID_EXPRY')
+    parser.add_argument('--senderid_list', help='SenderIDs to load to S3',
+                        type=int, default=[], action='append')
```

Comment:
It's a list of integers, where an empty list is valid. so yeah. If there's a different type declaration for that, I'm fine switching.